### PR TITLE
[MINI-5031] Improve test coverage

### PR DESCRIPTION
**Pull request by munir-rakuten**
# Description
After adding secure storage code coverage decreased a lot, this PR is for improving that situation. It is still not completed, We can improve more after we change the secure storage from file to database.

## Links
MINI-5031

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppSecureStorageDispatcher.kt [hunk 1]
```diff
@@ -11,12 +12,28 @@ import com.rakuten.tech.mobile.miniapp.storage.StorageState
...
-    private lateinit var miniAppId: String
-    private lateinit var secureStorage: MiniAppSecureStorage
-    private var cachedItems: Map<String, String>? = null
-    private var storageState: StorageState = StorageState.DEFAULT
+    @VisibleForTesting
+    internal lateinit var bridgeExecutor: MiniAppBridgeExecutor
+    @VisibleForTesting
+    internal lateinit var activity: Activity
+    @VisibleForTesting
+    internal lateinit var miniAppId: String
```
Comment (khairul-alam-licon) [inline miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppSecureStorageDispatcher.kt:20]: I think, `bridgeExecutor`, `activity` and `miniAppId` fields here could be private for now 🤔

File: miniapp/src/test/java/com/rakuten/tech/mobile/miniapp/js/MiniAppSecureStorageDispatcherSpec.kt [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.times
+import org.mockito.kotlin.verify
+
+@ExperimentalCoroutinesApi
+@RunWith(AndroidJUnit4::class)
+@Suppress("LargeClass")
+class MiniAppSecureStorageDispatcherSpec {
+    private val TEST_KEY = "key"
+    private val TEST_VALUE = "value"
```
Comment (khairul-alam-licon) [inline miniapp/src/test/java/com/rakuten/tech/mobile/miniapp/js/MiniAppSecureStorageDispatcherSpec.kt:32]: We can resolve this warning from AS:
Screen Shot 2022-05-19 at 14 46 21


So - we can move those fields into a companion object e.g.

```
private companion object {
        private const val TEST_KEY = "key"
        ...
}
```

File: miniapp/src/test/java/com/rakuten/tech/mobile/miniapp/js/MiniAppSecureStorageDispatcherSpec.kt [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.times
+import org.mockito.kotlin.verify
+
+@ExperimentalCoroutinesApi
```
Comment (khairul-alam-licon) [inline miniapp/src/test/java/com/rakuten/tech/mobile/miniapp/js/MiniAppSecureStorageDispatcherSpec.kt:27]: Do we need this annotation?

**Review by khairul-alam-licon — APPROVED** [review]
Thanks for recovering the coverage 🥇 
I have left few improvement suggestions.

**Review by gopi-karmakar — APPROVED** [review]

**Pull request merged**